### ISSUE TITLE
Settings: undef $custom_fields['ip_invoice_custom'] in settings page (ip-1.6.2-beta3 1st install)

### ISSUE THREAD

**Issue by sudwebdesign** [InvoicePlane/InvoicePlane#1163]
## Expected Behavior
Set the App

## Current Behavior
Only installed (without data) and go to the settings page
Impossible to save anything and navigate (Only return back with navigator `<-`)

<details>
  <summary><h3>Screenshot</h3></summary>

![Screenshot 2024-12-29 at 22-56-55 InvoicePlane-162b3-no-menu-on-settings](https://github.com/user-attachments/assets/bfaf82b1-ff90-4f18-b471-0581022dee67)

</details>

## Possible Solution
Where is defined `$custom_fields['ip_invoice_custom']` ?

## Steps to Reproduce
1. Install and login
2. go to `index.php/settings`

## Context (Environment)
New install without data

## Possible Implementation (Hard fix)
In `application/modules/email_templates/views/template-tags-invoices.php` [line 48](https://github.com/InvoicePlane/InvoicePlane/blob/bb5b8d2a58a26444afa696d13d8845f1c66ffa9e/application/modules/email_templates/views/template-tags-invoices.php#L48)
```php
            <?php foreach ($custom_fields['ip_invoice_custom'] as $custom) { ?>
```
Add if(isset())
```php
            <?php if(isset($custom_fields['ip_invoice_custom'])) foreach ($custom_fields['ip_invoice_custom'] as $custom) { ?>
```

## Failure Logs
See source (Ctrl+u)
you found an errors like: https://pastebin.com/TvDrhskr

**Comment by sudwebdesign** [comment]
I Found the menu: At end of page in `invoices` tab
![Screenshot 2024-12-29 at 22-56-55 InvoicePlane-162b3-menu-on-bottom-settings_invoices-tab](https://github.com/user-attachments/assets/06e17647-c5c0-4420-b4dc-20e0a4fe7d62)

**Comment by nielsdrost7** [comment]
Solved by [releasing version 1.6.2](https://github.com/InvoicePlane/InvoicePlane/releases/tag/v1.6.2)

**Comment by sudwebdesign** [comment]
Fixed by #1164 
Setting page showed normally (and QR showed in pdf invoice) :+1: 


But need to investigate why `$custom_fields['ip_invoice_custom']` is not set here.

And other little bug with this field `settings page` ->  `Invoices` tab -> `Enable Qr code` option -> `select#tags_invoice`
always set with `ID` (never saved (or selected?))
and (after added an Invoice custom field) this is not listed (TEXT field in my case).

@nielsdrost7 Maybe for IP-1.6.3 ;)

**Comment by nielsdrost7** [comment]
@sudwebdesign yep, I'll create the issue
